.github/workflows/helmsman_testrigs.yml

```
@@ -63,6 +71,19 @@ jobs:
             echo "HELMSMAN_MODE=${{ github.event.inputs.mode }}" >> $GITHUB_ENV
           fi        
 
+      - name: Set Profile
+        run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2086:info:2:56: Double quote to prevent globbing and word splitting

(shellcheck)